pkg/image/apiserver/admission/imagepolicy/imagepolicy.go

```diff
@@ -41,7 +41,37 @@ func Register(plugins *admission.Plugins) {
 			}
 			var config *imagepolicy.ImagePolicyConfig
 			if obj == nil {
-				config = &imagepolicy.ImagePolicyConfig{}
+				config = &imagepolicy.ImagePolicyConfig{
```

Comment:
This looks a little concerning.

@bparees I think this is part of the wiring of the imagepolicy admission plugin.  You should decide how you want to handle this.

/assign @bparees